### PR TITLE
Tighten persist interface for ExternalIOPlugin, use RFC6761 domains

### PR DESCRIPTION
**Pull request by Lekensteyn**

### CODE REVIEW

File: simp_le.py [hunk 1]
```diff
@@ -372,10 +371,11 @@ def get_output_or_fail(self, command):
     def persisted(self):
         """Call the external script and see which data is persisted."""
         output = self.get_output_or_fail('persisted')
+        persisted_keys = output.split('\n', 1)[0].split()
```
Comment (kuba) [inline simp_le.py]: Actually, I'd prefer current implementation to be as simple as possible, yet complying with the spec. So:
- this should be reverted to what it was before
- in future (post-#12 world, where `('key' in "account_key") is True`), we could just use `.split()`; in further future, if any kind of other metadata was passed on there, we could use your suggested code

File: simp_le.py [hunk 2]
```diff
@@ -338,15 +338,14 @@ class ExternalIOPlugin(OpenSSLIOPlugin):
 
     - whenever the script is called with `persisted` as the first
       argument, it should send to STDOUT a subset of three keywords:
-      `key`, `cart`, `chain` (in any order, using any separation
-      character);
+      `key`, `cart`, `chain` (in any order, separated by whitespace);
```
Comment (kuba) [inline simp_le.py]: I think we agreed to make keywords appear precisely on the first line of STDOUT, and separated by whitespace. Please make it explicit.

Comment (Lekensteyn) [inline simp_le.py]: Yes, I forgot to add this. Will do in the next iteration!

File: simp_le.py (hunk 1 above)
Comment (Lekensteyn) [inline simp_le.py]: I'll revert for now, backwards and forward compatibility is a bit hard to reach without the command and client being able to advertise the supported interface version.

**Comment by kuba** [comment]
Thank you very much for this PR and fixing review comments. Merging...

**Pull request merged**